### PR TITLE
add git ops files

### PR DESCRIPTION
**Pull request by ekwanNTT**
Add git ops files

### CODE REVIEW

**Review by sylar-zeyangz — CHANGES_REQUESTED** [review]
file names still have cdds :(

File: gitops/openshift/apps/automationtests-api/template/jag-adobe-common-service-tests.env [hunk 1]
```diff
@@ -0,0 +1,14 @@
+APP_NAME=jag-adobe-common-service
+COMPONENT_NAME=jag-adobe-common-service-tests
+OC_NAMESPACE=c71a03
```
Comment (sylar-zeyangz) [inline gitops/openshift/apps/automationtests-api/template/jag-adobe-common-service-tests.env:3]: to be updated when a oc namespace is created?

**Review by sylar-zeyangz — APPROVED** [review]
OC_NAMESPACE needs update when it's available. Others are good!

**Review by sylar-zeyangz — APPROVED** [review]

File: gitops/openshift/apps/automationtests-api/template/jag-adobe-common-service-tests.env (hunk 1 above)
Comment (ekwanNTT) [reply gitops/openshift/apps/automationtests-api/template/jag-adobe-common-service-tests.env:3]: Yes

**Review by sylar-zeyangz — APPROVED** [review]

**Pull request merged**